.github/workflows/abtesting.yml

```diff
@@ -35,6 +35,9 @@ jobs:
     runs-on: macos-11
     steps:
     - uses: actions/checkout@v2
+    - uses: mikehardy/buildcache-action@50738c6c77de7f34e66b870e4f8ede333b69d077
```

Comment:
This is 1.2.1. See https://github.com/mikehardy/buildcache-action/tags

Reply:
Yes, this is on purpose - https://github.com/firebase/firebase-ios-sdk/pull/9055#issuecomment-993510558